services/movie_session.py

```diff
@@ -42,3 +45,8 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list:
```

Comment:
Correct to this one

`return list(MovieSession.objects.get(id=movie_session_id).tickets.all().values("row", "seat"))`

Reply:
I didn't use this design because it's too long
services\movie_session.py:51:80: E501 line too long (98 > 79 characters)